src/translations/loadMessagesCatalog.tsx

```diff
@@ -0,0 +1,21 @@
+const fallbackLang = "en";
+
+const getLangCode = (code: string) =>
+  code.includes("-") ? code.split("-")[0] : code;
+
+export const lang = getLangCode(
+  (
+    (navigator.languages && navigator.languages[0]) ||
+    navigator.language ||
+    fallbackLang
+  ).toLowerCase()
+);
+
+export const loadCatalogs = async (language: string = lang) => {
+  const catalog = await import(
+     /* webpackMode: "lazy", webpackChunkName: "i18n-[index]" */
```

Comment:
Do we need this comment?

Reply:
As [the doc](https://lingui.js.org/guides/dynamic-loading-catalogs.html) says it's webpack’s `magic` comment.